.github/workflows/pr-check.yml

```diff
@@ -71,17 +71,10 @@ jobs:
         connection-string: '${{ secrets[matrix.connection_string_secret] }}Initial Catalog=${{ env.TEST_DB }};'
         path: ./__testdata__/testsql.sql
 
-    # Test that go-sqlcmd has been added to runner, this step will be removed in future PR when go-sqlcmd is hooked up
-    - name: Test go-sqlcmd is setup
-      run: sqlcmd -?
-
-    - name: Set database name for cleanup
-      if: always()
-      run: sed 's/$(DbName)/${{ env.TEST_DB }}/' ./__testdata__/cleanup.sql > ${{ runner.temp }}/cleanup.sql
-
     - name: Cleanup Test Database
       if: always()
       uses: ./
       with: 
         connection-string: '${{ secrets[matrix.connection_string_secret] }}Initial Catalog=master;'
-        path: ${{ runner.temp }}/cleanup.sql
+        path: ./__testdata__/cleanup.sql
+        arguments: '-v DbName="${{ env.TEST_DB }}"'
```

Comment:
This version fixes https://github.com/microsoft/go-sqlcmd/issues/114 which lets us use SQLCMD variables here again